src/docs/content.go

```diff
@@ -616,4 +613,29 @@ Enterprise Performance Computing (EPC)`
 	for all those blocks.`
 	VerifyExample string = `
   $ singularity verify container.sif`
+	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	// help
+	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	HelpUse   string = `help <command|image path>`
+	HelpShort string = `Display help for command||container if available`
```

Comment:
`command||container`
Change to either `command|container` _or_ `command or container image`